app/decorators/address_form_decorator.rb

```diff
@@ -1,4 +1,4 @@
-class AddressFormDecorator < BaseDecorator
+class AddressFormDecorator < BasePresenter
```

Comment:
Why not make this a `AddressFormPresenter` as well that way we aren't mixing concepts